README.md

```diff
@@ -8,9 +8,26 @@ Some new use cases for Web technology have motivated thinking about a multi-reso
 
+Local sharing tends to have persistent constraints on the amount of data
+transferred: each byte costs money. This means the client may be stuck with an
+outdated version of a package for a significant amount of time, including that
+package's vulnerabilities. It may be feasible to periodically check for:
+1. OCSP notification that a package's certificate has been revoked, and
+2. A yet-to-be-designed notification that the package is critically vulnerable
+   and needs to be disabled until it can be updated.
+
 ### Physical Web
 [Beacons](https://google.github.io/physical-web/) and other physical web devices often want to 'broadcast' various content locally. Today, they broadcast a URL and make the user's device go to a web site. This delivers the trusted content to the user's browser (user can observe the address bar to verify) and allow web apps to talk back to their services. It can be useful to be able to broadcast a package containing several pages or even a simple web app, even without a need to immediately have a Web connection - for example, via Bluetooth. If combined with signature from the publisher, the loaded pages may be treated as if they were loaded via TLS connection with a valid certificate, in terms of the [origin-based security model](https://tools.ietf.org/html/rfc6454). For example, they can use [`fetch()`](https://fetch.spec.whatwg.org/#fetch-api) against its service or use "Add To Homescreen" for the convenience of the user.
 
+Physical web beacons may be located in a place that has no connectivity, and
+their owner may only visit to update them as often as their battery needs
+replacement: annually or less often. This leaves a large window during which a
+certificate could be compromised or a package could get out of date. We think
+there's no way to prevent a client from trusting its initial download of a
+package signed by a compromised certificate. When the client gets back to a
+public network, it should attempt to validate both the certificate and the
+package, likely by fetching the package's `Content-Location`.
```

Comment:
This makes me think that analogue to the HEAD request where the client could fetch only the manifest/index and revalidate stale portions would be useful.